.github/dependabot.yaml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
We will more than likely use Forking Renovate for dependencies, but we'll need to determine where it will actually be useful given the inter dependencies of a lot AI/ML packages.

Reply:
Ok. Removed. We can configure Renovate later. Eventually, we might want to enable on-demand PR creation to reduce noise.